test/data/valid_instructions_with_invalid_class.yml

```diff
@@ -0,0 +1,5 @@
+---
+:class: UnknownClass
```

Comment:
There's a slightly mismatch here between the name of the file and the name of the class. I don't think there's really much need for that, and if you harmonised them it could save someone later from trying to puzzle out whether the difference is important.